test/services/jskeymanager.ts

```diff
@@ -132,6 +132,15 @@ export class JsKeyManager implements KeyManager {
 
             const keyStore = this.keyStorage[options.level];
 
+            if (options.level === KeyManagerLevel.PIN) {
```

Comment:
remove this challenge guarde completly

Reply:
https://github.com/Tonomy-Foundation/Tonomy-ID/issues/396

Reply:
ok, as discussed. we will create a new function on the KeyManager
`checkKey(keyManagerLevel: KeyManagerLevel, challenge): Promise<boolean>`
which gets implemented in the instances (tests, TID and App Websites) to check if the challenge is correct

Reply:
this is used in checkPin()